pkgs/applications/audio/mpg123/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv
 , fetchurl, alsaLib
-, buildPlatform, hostPlatform
+, hostPlatform
```

Comment:
@Ericson2314 ```The "target platform" is black sheep. The other two intrinsically apply to all compiled software—or any build process with a notion of "build-time" followed by "run-time". The target platform only applies to programming tools, and even then only is a good for for some of them.``` Please remove the extra "for" here and make it "a black sheep" or simply say "is the exception", since you are now assuming that your audience is aware of the finer details of the English language without benefit.

Otherwise, I am happy to see documentation being written.